src/main/java/pl/most/typer/model/matches/Score.java

```diff
@@ -15,13 +16,15 @@
     private String winner;
     private String duration;
 
-    @OneToOne
+    @OneToOne(cascade = CascadeType.ALL)
     private TeamGoals fullTime;
-    @OneToOne
+    @OneToOne(cascade = CascadeType.ALL)
     private TeamGoals halfTime;
-    @OneToOne
+    @OneToOne(cascade = CascadeType.ALL)
     private TeamGoals extraTime;
-    @OneToOne
+    @OneToOne(cascade = CascadeType.ALL)
     private TeamGoals penalties;
 
+    @OneToOne(cascade = CascadeType.ALL)
```

Comment:
Tutaj cascade jest chyba nie potrzebny -> Cascade dziala wtedy w dwie strony. Jak zapiszesz Match to zapiszesz Score, ale również jak usuniesz Score to usuniesz Match całkowicie. Do przemyślenia

Reply:
done